Polyfills/XMLHttpRequest/Source/XMLHttpRequest.cpp

```diff
@@ -149,6 +151,17 @@ namespace Babylon::Polyfills::Internal
         m_request.ResponseType(ResponseType::StringToEnum(value.As<Napi::String>().Utf8Value()));
     }
 
+    Napi::Value XMLHttpRequest::GetResponseHeader(const Napi::CallbackInfo& info)
+    {
+        const auto headerName{info[0].As<Napi::String>().Utf8Value()};
+        return Napi::Value::From(Env(), m_request.ResponseHeader(headerName));
```

Comment:
This implementation is incorrect.

[Return value](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/getResponseHeader#return_value)

> A string representing the header's text value, or null if either the response has not yet been received or the header doesn't exist in the response.

We have to return null in some cases.